consensus/poet/core/tests/test_consensus/test_poet_block_publisher.py

```diff
@@ -674,3 +674,233 @@ def test_block_publisher_success_case(
         self.assertTrue(
             block_publisher.initialize_block(
                 block_header=mock_block.header))
+
+    @patch('sawtooth_poet.poet_consensus.wait_timer.WaitTimer')
```

Comment:
This should be `@patch('sawtooth_poet.poet_consensus.poet_block_publisher.WaitTimer')`.  Should always patch where the class/module is being imported into, now where it is being imported from.